README.md

```diff
@@ -166,3 +169,5 @@ Icons are also available as a [Vue package](https://github.com/mainvest/vue-simp
 ### WordPress
 
 Icons are also available as a [WordPress plugin](https://wordpress.org/plugins/simple-icons/) created by  [@tjtaylo](https://github.com/tjtaylo).
+
+[icon slug]: ./slugs.md
```

Comment:
This link will be broken in the NPM page, should be absolute, right?

```suggestion
[icon slug]: https://github.com/simple-icons/simple-icons/blob/develop/slugs.md
```

Reply:
From my experience NPM converts these kinds of URLs correctly. I propose we try it like this (for the reasons below) and change it in the next release if indeed it doesn't work on NPM (or Packagist) - in which case I think it should point to `master`, not `develop`.

For the record, I prefer the local URL as 1) it doesn't need to be changed if the base URL or remote changes and 2) it works locally.

Reply:
Oh, I didn't know this. It seems that [Packagist also converts relative links to absolute ones](https://github.com/composer/packagist/blob/main/src/Package/Updater.php#L691), so I'm fine with this :+1: